### PR TITLE
Update Known Instance in InnerSource Hackathon pattern

### PR DESCRIPTION
**Pull request by spriya-m**

### CODE REVIEW

**Review by spier — APPROVED** [review]

**Comment by spier** [comment]
Amazing to hear that this pattern is in use at IKEA!! Thank you for contributing this @spriya-m!

Just curious: was it difficult to make this info publicly shareable?

**Comment by spriya-m** [comment]
Thank you @spier. It wasn't difficult, but had to wait for the formal approval.

**Pull request merged**
